hibernate-reactive-core/src/test/java/org/hibernate/reactive/BaseMutinyTest.java

```diff
@@ -0,0 +1,104 @@
+package org.hibernate.reactive;
+
+import io.smallrye.mutiny.Uni;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.Timeout;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.hibernate.boot.registry.StandardServiceRegistry;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.AvailableSettings;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.reactive.containers.PostgreSQLDatabase;
+import org.hibernate.reactive.mutiny.Mutiny;
+import org.hibernate.reactive.service.ReactiveConnection;
+import org.hibernate.reactive.service.ReactiveConnectionPool;
+import org.hibernate.reactive.stage.Stage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.CompletionStage;
+
+@RunWith(VertxUnitRunner.class)
+public abstract class BaseMutinyTest {
+
+	@Rule
+	public Timeout rule = Timeout.seconds( 3600 );
+
+	private Mutiny.Session session;
+	private ReactiveConnection connection;
+	private org.hibernate.SessionFactory sessionFactory;
+	private ReactiveConnectionPool poolProvider;
+
+	protected static void test(TestContext context, CompletionStage<?> cs) {
```

Comment:
I would expect a mutiny test to use `Uni` here, but :shrug: :)

Reply:
Yes yes, at this point I had not finished work on the test harness but it is finished now, so if you would be looking at the current code, and not a 4-day-old pull request, you wouldn't need to see things like this :-P

Reply:
Well, you raise the PR while I'm on PTO and I'm kind enough to review it even when you didn't wait for me to merge it, so don't complain too much ;)